lib/openstack_query/runners/server_runner.py

```diff
@@ -98,13 +101,14 @@ def _run_query_on_project(
         """
         server_filters = {"project_id": project["id"], "all_tenants": True}
         server_filters.update(filter_kwargs if filter_kwargs else {})
-        return list(conn.compute.servers(filters=server_filters))
+        return list(conn.compute.servers(all_projects=False, **server_filters))
```

Comment:
Future issue, we should allow someone to pass `all_project=True` for some of the st2 queries

Reply:
no it will always be false here - since were going through each project to find the servers
we can do `all_project=True` instead of iterating through projects but we run into a bug where openstack can't list more than 1000 VMs